mobilecoind/strategies/requirements.txt

```diff
@@ -10,7 +10,7 @@ grpcio==1.44.0
 grpcio-tools==1.44.0
 idna==3.3
 ipdb==0.12.2
-ipython==8.10.0
```

Comment:
Looks like one of these branches is out-of-date with master...

Reply:
I think this actually causes a [failure](https://github.com/mobilecoinfoundation/mobilecoin/actions/runs/4255320061/jobs/7402819293) in the build process

Reply:
Please open an issue so we can resolve this, don't revert dependency updates.

Reply:
https://github.com/mobilecoinfoundation/mobilecoin/issues/3207